governance-charter.md

```diff
@@ -109,11 +109,9 @@ vote in the elections, and may choose to run in the 2021 election).
 
 ### Members of Standing
 
-Standing in the OpenTelemetry community will be defined by the union of:
-
-- Active members - those with 20 contributions (PRs, Issues, Comments etc.) in
-  the prior rolling year.
-- All approvers, and maintainers in the OpenTelemetry organization
+Standing in the OpenTelemetry community will be defined as the set of all
```

Comment:
what's the motivation to remove "active" part of it? There is always an "exceptions" excel table

Reply:
Because it's now encapsulated in the opentelemetry org membership criteria. Why have two sets of standards for membership?

Reply:
I don't follow. There is nothing about removing people from being members of an organization. And 20 active contributions like reporting an issue is a way to see if a member is still active

Reply:
we should... maybe have one process, including a process for removing people from the org membership then, rather than trying to maintain two parallel processes? or just accept that there may be inactive past contributors, who might choose to vote (but are unlikely to do so if they've already gone inactive)

Reply:
we discussed that when was writing the document. At a time we decided that 20 contributions rule is good enough until the process of de-membershipment will be figured out. In my opinion the scope of this PR does not include this change.

Reply:
Happy to approve the 3month -> 1 month change, but this question needs a bit more thinking.

Reply:
Okay, I'll drop this voting change as it's less pressing.

Reply:
Sounds good. We can discuss it further later.

Reply:
Dropped, please remove blocking objection.